### PR TITLE
FOGL-8803: Fixed Deletion of persisted data in Developer mode

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts [hunk 1]
```diff
@@ -14,8 +14,10 @@ export class ConfigurationGroupComponent implements AfterViewInit {
 
   selectedGroup = 'Basic';
   @Input() category;
-  groups = [];
+  @Input() plugin;
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts:17]: Should we group all @Input together?

Input
Ouput
Default assignments

File: src/app/components/common/node-editor/node-editor.component.ts [hunk 2]
```diff
@@ -1078,6 +1082,16 @@ export class NodeEditorComponent implements OnInit {
         });
   }
 
+  getServiceStatus() {
+    if (this.from == 'south') {
+      const service: Service = this.services.find(service => service.name === this.serviceName);
+      return service.status;
+    } else if (this.from == 'north') {
+      const task: NorthTask = this.tasks.find(task => task.name === this.serviceName);
```
Comment (praveen-garg) [inline src/app/components/common/node-editor/node-editor.component.ts:1090]: North can be a service as well.

Comment (mshariq-nerd) [reply src/app/components/common/node-editor/node-editor.component.ts:1090]: We are good here. In task we are getting service/task from north side.

**Review by MonikaSharma06 — APPROVED** [review]

File: src/app/components/common/node-editor/node-editor.component.ts (hunk 2 above)
Comment (praveen-garg) [reply src/app/components/common/node-editor/node-editor.component.ts:1090]: Add inline comment please

**Review by hemantgarg923 — APPROVED** [review]

File: src/app/components/common/node-editor/node-editor.component.ts (hunk 2 above)
Comment (mshariq-nerd) [reply src/app/components/common/node-editor/node-editor.component.ts:1090]: I will take care of this in other PR.

**Pull request merged**